components/Modal.tsx

```diff
@@ -50,36 +50,61 @@ const Modal: FC<{
                             <span className={styles.line} />
                         </button>
                         <span className={`flex flex-col ${styles.text}`}>
-                            <a
-                                className='text-blue-700 hover:underline my-3'
-                                href={`https://solscan.io/account/${account}${
-                                    connection.rpcEndpoint.includes('devnet') ? '?cluster=devnet' : ''
-                                }`}
-                                target='_blank'
-                                rel='noopener noreferrer'
-                            >
-                                View in Solscan
-                            </a>
+                            <div className='mt-5'>
+                                <a
+                                    className='text-blue-700 hover:underline my-3 mt-4 mr-2'
+                                    href={`https://solscan.io/account/${account}${
+                                        connection.rpcEndpoint.includes('devnet') ? '?cluster=devnet' : ''
+                                    }/`}
```

Comment:
adding the `/` actually does not fix the issue, but makes it a bit worst since the shown address is in the wrong cluster.